mysite/lms/views.py

```diff
@@ -73,6 +73,10 @@ def get(self, request, course_id):
             request, 'lms/course_page.html',
             dict(
                 course=course,
+                course_author=(
```

Comment:
why not to do this in templates? 
In general there's no difference to do it here or in template, but why should we pass more context to templates if it's not required?

```
{% if course.addedBy.get_full_name %}
{{ course.addedBy.get_full_name }} 
{% else %} 
{{ course.addedBy.username }} 
{% endif %}
```

It will not require any changes in views and will be displayed on all pages not depending on passed context (for example we add new page, which will use one of provided templates. In this case we have to remember to pass this parameter to the template to show it correctly, that's no good as for me.)